README.md

```diff
@@ -1,5 +1,7 @@
 # pfSense
 
+peterberbec mods to support OCSP must-staple
```

Comment:
You'll have to remove these changes to README.md, it's trying to merge them into the pfSense repo, they aren't just local to you.

Reply:
That was to ensure I was looking at the right repo locally, i'l delete it obviously!